utils/m0reportbug

```diff
@@ -651,7 +655,7 @@ binaries() {
 
     mkdir -p $outdir
     ## Note, that we intentionally skip `m0tr.ko' as it's too big (42M).
-    for f in m0mkfs m0d m0ham m0ut m0beck m0betool ${libs[@]}; do
+    for f in m0mkfs m0d m0ham m0ut m0beck m0betool m0client m0touch m0cp m0cat ${libs[@]}; do
```

Comment:
Please update the history part.

### History:
###
### Date       Version Description